assets/scss/base/_buttons.scss

```diff
@@ -32,6 +33,24 @@ button, .button {
   @include core-19();
 }
 
+.button--blue {
+  @extend .button--get-started;
+  @include button($govuk-blue-colour);
+  @include bold-27();
+  width:100%;
+  padding-left:3em;
+  padding-top:0.65em;
+  height:2.2em;
+  box-sizing:border-box;
+}
+
+.button--blue:before {
+  background: url('../images/calendar.png');
```

Comment:
The addition of a calendar icon shouldn't be specific to buttons. This could be

`.icon--calendar` and live in `base/_icons.scss`